.travis.yml

```diff
@@ -15,50 +37,57 @@ env:
     - xTHREADING=1 xMPI=0 xGSL=0 xLTDL=1 xREADLINE=1 xPYTHON=0 xMUSIC=0 xSTATIC_ANALYSIS=1 CACHE_NAME=JOB   # only threading
     - xTHREADING=1 xMPI=1 xGSL=1 xLTDL=1 xREADLINE=1 xPYTHON=1 xMUSIC=1 xSTATIC_ANALYSIS=1 CACHE_NAME=JOB   # full
 matrix:
-  # do notify immediately about it when a job of a build fails.
+  # Do notify immediately about it when a job of a build fails.
   fast_finish: true
 cache:
   directory:
     - $HOME/.cache
 
 before_install:
+  - echo "+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +"
+  - echo "+         P R E P A R E   N E S T   B U I L D   E N V I R O N M E N T         +"
+  - echo "+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +"
   - export SOURCEDIR=$PWD
   - cd $HOME/build
   - echo $PATH
 
-  # update package repository status (-qq is more quiet)
+  # Update package repository status (-qq is more quiet).
   - sudo rm -rf /var/lib/apt/lists/*
   - ls /etc/apt/sources.list.d/
   - sudo apt-get update -qq
 
-  # remove any obsolete libraries
+  # Remove any obsolete libraries.
```

Comment:
Change to `# Remove automatically all unused package`

(this is the text, which the the help for `apt-get autoremove` prints)